### ISSUE TITLE
tons of console errors

### ISSUE THREAD

**Issue by DavidFW1960** [RomRider/apexcharts-card#17]
![image](https://user-images.githubusercontent.com/36390762/106214489-fabc8100-6222-11eb-936b-b5769499f73d.png)

Not sure now... Just did an empty cache/hard reload and now I see no errors....

**Comment by RomRider** [comment]
Yes, that is a bug in ApexCharts, I can't do anything about it unfortunately :(. You can ignore them.
It happens mainly when you have `null` data in the time line.

**Comment by k1ng440** [comment]
add an option to delete null points

**Comment by andrewdolphin** [comment]
I'm in the latest version and still seeing these errors (10's of thousands of them!)

**Comment by RomRider** [comment]
It happened again also on my side. It's not breaking anything, so I'll keep this opened. Maybe it will solve itself in the future while upgrading the library.

**Comment by Mariusthvdb** [comment]
just found this and updated and yes, I do believe the errors have disappeared ;-)

thx!

**Comment by RomRider** [comment]
Cool thanks for the feedback :)

**Comment by Mariusthvdb** [comment]
right. I ofc shouldn't have been so quick....

<img width="947" alt="Schermafbeelding 2022-04-22 om 10 21 38" src="https://user-images.githubusercontent.com/33354141/164648598-b0c3db13-7e0d-405d-b839-cd66b9c2c090.png">

and the odd thing is, this happens on pages even without a single apexchart card...

so sorry

**Comment by arboeh** [comment]
Same here with v2.0.4
update_interval is 1min or 15min.

![2023-06-07 07_02_48-Übersicht – Home Assistant](https://github.com/RomRider/apexcharts-card/assets/12322114/ccec0bf9-5b6f-48a8-95b9-dc042a7e5711)

**Comment by kvanbiesen** [comment]
made a dirty hack to get rid of most of the spam Log. Only drops an console error when leaving a page that contains an apex card.

/www/communty/apexchart-card/apexchartcard.js
Delete the GZ file and edit ABOVE file

!!!!!!!!!!!!!!!!MAKE BACKUP OF THE FILE!!!!!!!!!!!!!!!!!!!

Use a unminify tool like to make it more readable: https://www.unminify2.com/


-----------------
Line: 8661:
replace 
 for (var o in t) t.hasOwnProperty(o) && e.setAttribute(o, t[o])
with
 for (var o in t) t.hasOwnProperty(o) && !String(t[o]).includes("NaN") && e.setAttribute(o, t[o])


Line: 23394:
Insert new line before "stroke-width" == e ? this.attr("stroke",......................
containing 
if (String(t).includes("NaN")) return null;
---------------------------

Save and do a hard refresh (this is hardest part), i had to alter resources under settings- dashboard and change the number in the back of the url.

**Comment by kvanbiesen** [comment]
Btw for those not comfortable editing the JS file.

Attached here is a copy, replace in /homeassistant/www/community/apexcharts-card/apexcharts-card.js (rename remove .txt extension)
[apexcharts-card.js.txt](https://github.com/user-attachments/files/17042357/apexcharts-card.js.txt)


WARNING its not minimized so slightly bigger than orginal (minify yourself)

Reduces the spam a lot.

**Comment by haysdb** [comment]
Thank you for the file. Now I only see 4 errors and not hundreds.

**Comment by kvanbiesen** [comment]
Yeah I know. It's not fixed fixed but reduces the spam. Maybe someone with
more Js knowledge can fix it better
